src/electron/auto-update/auto-updater-client.ts

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import log from 'electron-log';
 
 export class AutoUpdaterClient {
-    constructor(private readonly autoUpdater) {}
+    constructor(private readonly autoUpdater) {
```

Comment:
is there any strong reason to not have a type for `autoUpdater`? looks like `AppUpdater` is the right type to use

Reply:
I didn't notice the missing type, I agree with you - changed